test/Confluent.Kafka.IntegrationTests/Tests/AssignOverloads.cs

```diff
@@ -28,12 +28,12 @@ public static partial class Tests
         /// <summary>
         ///     Simple test of both Consumer.Assign overloads.
         /// </summary>
-        [Theory, MemberData(nameof(KafkaParameters))]
+        [Theory, ClassData(typeof(KafkaParameters))]
         public static void AssignOverloads(string bootstrapServers, string topic)
         {
             var consumerConfig = new Dictionary<string, object>
             {
-                { "group.id", "test-consumer-group" },
+                { "group.id", "assign-overloads-cg" },
```

Comment:
every test now uses a different consumer group id because if a test fails, subsequent tests with the same consumer group will not be able to join the group because it was often still be alive from the failed test (session.timeout.ms min = 6000 is a relatively long time).